### PR TITLE
target: Add JHEMCU F722 target board support

### PR DESCRIPTION
**Pull request by jhemcu**
Add support for JHEMCU F722 target

### CODE REVIEW

**Comment by mmosca** [comment]
Board has been received by me and @shota3527

File: src/main/target/JHEMCUF722/target.c [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+#include "drivers/sensor.h"
+
+BUSDEV_REGISTER_SPI_TAG(busdev_bmi270,   DEVHW_BMI270,   BMI270_SPI_BUS,   BMI270_CS_PIN,   NONE,   0, DEVFLAGS_NONE, IMU_BMI270_ALIGN);
+BUSDEV_REGISTER_SPI_TAG(busdev_icm42688, DEVHW_ICM42605, ICM42605_SPI_BUS, ICM42605_CS_PIN, NONE, 0, DEVFLAGS_NONE, IMU_ICM42605_ALIGN);
+BUSDEV_REGISTER_SPI_TAG(busdev_mpu6000,  DEVHW_MPU6000,   MPU6000_SPI_BUS, MPU6000_CS_PIN,  NONE,  0, DEVFLAGS_NONE, IMU_MPU6000_ALIGN);
+
+timerHardware_t timerHardware[] = {
+    DEF_TIM(TIM9,  CH2, PA3,  TIM_USE_PPM,   0, 0 ), //PPM&SBUS 
+
+    DEF_TIM(TIM3,  CH3, PB0,  TIM_USE_MC_MOTOR | TIM_USE_FW_MOTOR, 0, 0),  // S2 DMA1_S7_CH5
```
Comment (mmosca) [inline src/main/target/JHEMCUF722/target.c]: Please replace the servo usage and motor flags with TIM_USE_OUTPUT_AUTO to comply with INAV 7.0 changes

Comment (jhemcu) [reply src/main/target/JHEMCUF722/target.c]: Done.

File: src/main/target/JHEMCUF722/target.h [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+#define USE_TARGET_IMU_HARDWARE_DESCRIPTORS
+
+#define USE_SPI
+#define USE_SPI_DEVICE_1
+#define SPI1_SCK_PIN         PA5
+#define SPI1_MISO_PIN        PA6
+#define SPI1_MOSI_PIN        PA7
+
+#define USE_IMU_ICM42605
+#define IMU_ICM42605_ALIGN   CW90 DEG
```
Comment (mmosca) [inline src/main/target/JHEMCUF722/target.h]: You are missing a _ here. It should be CW90_DEG, without the space.

Comment (Linjieqiang) [reply src/main/target/JHEMCUF722/target.h]: Good catch

File: src/main/target/JHEMCUF722/target.h [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+#define SPI1_MOSI_PIN        PA7
+
+#define USE_IMU_ICM42605
+#define IMU_ICM42605_ALIGN   CW90_DEG
+#define ICM42605_SPI_BUS     BUS_SPI1
+#define ICM42605_CS_PIN      PA4
+#define ICM42605_EXTI_PIN    PC3   
+
+#define USE_IMU_BMI270
+#define IMU_BMI270_ALIGN     CW180_DEG
```
Comment (mmosca) [inline src/main/target/JHEMCUF722/target.h]: Is this supposed to be CW90 or CW180?

File: src/main/target/JHEMCUF722/target.h [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+#define BMI270_SPI_BUS       BUS_SPI1
+#define BMI270_CS_PIN        PA4
+#define BMI270_EXTI_PIN      PC3   
+
+#define USE_IMU_MPU6000
+#define IMU_MPU6000_ALIGN    CW180_DEG
```
Comment (mmosca) [inline src/main/target/JHEMCUF722/target.h]: Is this supposed to be CW90 or CW180?

**Comment by jhemcu** [comment]
Thanks.

**Comment by dev0stranger** [comment]
Hello,  @jhemcu

**Pull request merged**